kani-driver/src/session.rs

```diff
@@ -64,6 +64,31 @@ impl KaniSession {
         })
     }
 
+    /*
+    Setup the default version of cargo being run, based on the type/mode of installation for kani
+    If kani is being run in developer mode, then we use the one provided by rustup as we can assume that the developer will have rustup installed
+    For release versions of Kani, we use a version of cargo that's in the toolchain that's been symlinked during `cargo-kani` setup. This will allow
+    Kani to remove the runtime dependency on rustup later on.
+    */
+    pub fn setup_cargo_command(&self) -> Result<Command> {
```

Comment:
This function doesn't really need to be part of KaniSession. If you move it out, you don't need to duplicate the code.